.gitignore

```diff
@@ -58,3 +58,7 @@ a.out
 *.pb.h
 *.pb.cc
 *_pb2.py
+
+# Compiled executables used for docker build
+/docker/presto-cli-*-executable.jar
```

Comment:
Can we just exclude everything in `/docker`?

Reply:
I think we can't exclude everything in the `/docker` folder. An entry point script, a dockerfile and a few other files already exist in this folder. I have added to ignore the files that get added as part of the testing framework.